code/controllers/subsystem/playtime.dm

```diff
@@ -0,0 +1,305 @@
+/**
+ * playtime tracking system
+ */
+SUBSYSTEM_DEF(playtime)
```

Comment:
this is the most derranged and wasteful way i have ever seen someone impliment a playtime tracker